### ISSUE TITLE
Incorrect require-accessible-name error: label and input element before eRuby block

### ISSUE THREAD

**Issue by yonta** [markuplint/markuplint#1147]
- Markuplint version: `3.12.1`
- Parser lang: ERB
- Node.js version: `18.17.1`
- OS: Windows11 with WSL2 Ubuntu 22.04.02 LTS

## Describe the bug

Label element and input element which has correct accessible name before eRuby block is occure require-accessible-name error.

## Code Example

```erb
    <label for="cheese">Do you like cheese?</label>
    <input type="checkbox" id="cheese">
    <% pp "anything" %>
```

```json
// .markuplintrc
{
  "extends": ["markuplint:recommended"],
  "parser": {
    "\\.erb$": "@markuplint/erb-parser"
  }
}
```

## Steps To Reproduce

1. run markuplint

## The current behavior

require-accessible-name error

```console
$ yarn markuplint test.html.erb
<markuplint> error: アクセシブルな名前が必要です (require-accessible-name) /home/kei/git/sakazuki/test.html.erb:10:5
  9: ••••<label•for="cheese">Do•you•like•cheese?</label>
  10: ••••<input type="checkbox" id="cheese">
  11: ••••<%•pp•"anything"•%>
```

## The expected behavior

not errored

**Comment by YusukeHirao** [comment]
@yonta Thank you for this report.

I'll check it.

**Comment by yonta** [comment]
Sorry, the current behavior output was wrong, I corrected it.
